e2e-tests/insurance/cypress/e2e/journeys/export-contract/agent-charges/agent-charges.spec.js

```diff
@@ -116,10 +118,10 @@ context(
       });
 
       describe(`when submitting with ${METHOD} as ${FIXED_SUM}`, () => {
-        it(`should redirect to ${CHECK_YOUR_ANSWERS}`, () => {
+        it(`should redirect to ${AGENT_CHARGES_CURRENCY}`, () => {
           cy.completeAndSubmitAgentChargesForm({ fixedSumMethod: true });
 
-          cy.assertUrl(checkYourAnswersUrl);
+          cy.assertUrl(agentChargesCurrencyUrl);
         });
 
         // TODO: EMS-3828 - renable
```

Comment:
is there a reason this is not removed here but is in other places?

Reply:
Yes - it's been removed from other places where we no longer need to skip, because the user flow has changed